pages/support.tsx

```diff
@@ -1,10 +1,12 @@
 import BottomNav from '../components/BottomNav'
 import styles from '../styles/Home.module.css'
+import TopNav from '../components/TopNav'
 
 export default function Support() {
   return (
     <div className={styles.container}>
       {}
+      <TopNav pageName={'Support'} displayBackButton={true} />
```

Comment:
I think I would actually just comment here that this is `true` for demonstration purposes but should be false in a `//TODO: ` comment.